sympy/combinatorics/fp_groups.py

```diff
@@ -76,6 +76,27 @@ def __new__(cls, fr_grp, relators):
     def free_group(self):
         return self._free_group
 
+    @property
+    def relators(self):
+        return tuple(self._relators)
```

Comment:
I think you could do that. For `relators` in particular, that seems better than defining a property. But it would probably be preferable to do the changes in a separate PR.